experiment/measurer.py

```diff
@@ -67,13 +68,158 @@ def exists_in_experiment_filestore(path: pathlib.Path) -> bool:
+    with multiprocessing.Pool() as pool, multiprocessing.Manager() as manager:
+        q = manager.Queue()  # pytype: disable=attribute-error
+        try:
+            covered_regions = get_all_covered_region(experiment, pool, q)
+            json_src_dir = get_experiment_folders_dir()
+            json_src = os.path.join(json_src_dir, 'covered_regions.json')
+            with open(json_src, 'w') as src_file:
+                json.dump(covered_regions, src_file)
+            json_dst = exp_path.filestore(json_src)
+            filestore_utils.cp(json_src, json_dst, expect_zero=False)
+        except Exception:  # pylint: disable=broad-except
+            logger.error('Error occurred during coverage measuring.')
+    logger.info('Finished storing coverage data')
+
+
+def get_all_covered_region(experiment: str, pool, q) -> dict:
+    """Get regions covered for each pair for fuzzer and benchmark."""
+    logger.info('Measuring all fuzzer-benchmark pairs for final coverage data.')
+
+    benchmarks = get_all_benchmark_names(experiment)
+    fuzzers = get_all_fuzzer_names(experiment)
+
+    get_covered_region_args = [(experiment, fuzzer, benchmark, q)
+                               for fuzzer in fuzzers
+                               for benchmark in benchmarks]
+
+    result = pool.starmap_async(get_covered_region, get_covered_region_args)
+
+    # Poll the queue for covered region data and save them in a dict until the
+    # pool is done processing each combination of fuzzers and benchmarks.
+    all_covered_regions = {}
+
+    while True:
+        try:
+            covered_regions = q.get(timeout=COV_DIFF_QUEUE_GET_TIMEOUT)
+            all_covered_regions.update(covered_regions)
+        except queue.Empty:
+            if result.ready():
+                # If "ready" that means pool has finished. Since it is
+                # finished and the queue is empty, we can stop checking
+                # the queue for more covered regions.
+                logger.debug(
+                    'Finished call to map with get_all_covered_region.')
+                break
+
+    eliminate_set(all_covered_regions)
+    logger.info('Done measuring all differential data.')
+    return all_covered_regions
+
+
+def get_covered_region(experiment: str, fuzzer: str, benchmark: str,
+                       q: multiprocessing.Queue):
+    """Get the final covered region for a specific pair of fuzzer-benchmark."""
+    initialize_logs()
+    logger.debug('Measuring covered region: fuzzer: %s, benchmark: %s.', fuzzer,
+                 benchmark)
+    key = get_fuzzer_benchmark_key(fuzzer, benchmark)
+    covered_regions = {key: set()}
+    try:
+        trial_ids = get_trial_ids(experiment, fuzzer, benchmark)
+        for trial_id in trial_ids:
+            logger.info('Measuring covered region: trial_id = %d.', trial_id)
+            summary_file = get_summary_file_path(fuzzer, benchmark, trial_id)
+            with open(summary_file) as summary:
```

Comment:
Create a helper function for
            with open(summary_file) as summary:
                coverage_info = json.loads(summary.readlines()[-1])

create another one for

	                functions_data = coverage_info["data"][0]['functions']
                for function_data in functions_data:
                    for region in function_data["regions"]:
                        if region[4] != 0 and region[-1] == 0:
                            # region[4] indicates if it is hit.
                            # region[-1] indicates if it a 'code region'.
                            covered_regions[key].add(tuple(region[:4]))